staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -666,7 +666,16 @@ func (s preparedGenericAPIServer) NonBlockingRun(stopCh <-chan struct{}, shutdow
 }
 
 // installAPIResources is a private method for installing the REST storage backing each api groupversionresource
-func (s *GenericAPIServer) installAPIResources(apiPrefix string, apiGroupInfo *APIGroupInfo, openAPIModels *spec.Swagger) error {
+func (s *GenericAPIServer) installAPIResources(apiPrefix string, apiGroupInfo *APIGroupInfo, openAPIModels map[string]*spec.Schema) error {
+	var typeConverter fieldmanager.TypeConverter
+
+	if len(openAPIModels) > 0 {
```

Comment:
Why are we doing this? I'm not sure how this would be relevant (that it's non-nil but empty), but that seems weird.